mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.DeleteOrder = (req, res) => {
+  Order.remove({ _id: req.params.orderId }) 
+    .exec((error, result) => {
+      if (error) {
```

Comment:
Inconsistent error handling. In some places, you're returning `res.status(400).json({ error: errorHandler(err) })`, while in others, you're returning `res.status(400).json({ error: 'Error message' })`. Choose a consistent approach throughout the codebase.